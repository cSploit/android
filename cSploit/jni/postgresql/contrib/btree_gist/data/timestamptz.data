\N
2002-08-30 15:27:57 GMT
1997-11-16 06:01:36 GMT-2
\N
2010-04-02 01:21:20 GMT+2
2019-09-21 05:54:01 GMT+6
1978-03-04 09:31:20 GMT-9
1992-11-08 19:54:37 GMT-4
2002-02-11 19:06:34 GMT-1
2012-06-06 18:35:22 GMT+3
2012-11-23 22:44:32 GMT+3
1997-09-27 12:43:45 GMT-2
1972-12-18 04:24:22 GMT-11
\N
2028-09-29 22:06:36 GMT+9
1976-03-31 15:03:19 GMT-10
1996-06-26 06:20:33 GMT-3
2014-07-06 05:37:36 GMT+4
2032-05-21 18:59:04 GMT+10
2018-12-18 10:59:54 GMT+3
1986-02-15 10:47:42 GMT-6
1974-06-02 17:04:29 GMT-10
2020-11-03 05:11:44 GMT+6
2024-09-29 08:51:37 GMT+7
1996-07-03 02:47:04 GMT-3
1973-02-11 11:17:51 GMT-11
2019-12-02 01:36:35 GMT+6
2035-09-04 07:23:02 GMT+11
\N
2018-05-16 21:07:23 GMT+5
\N
2030-05-04 03:37:48 GMT+9
2002-10-27 17:43:22 GMT
2036-03-04 08:06:10 GMT+11
2002-10-29 03:21:13 GMT
2000-06-22 14:23:04 GMT-1
1981-01-01 00:15:56 GMT-8
2001-12-31 12:30:28 GMT-1
2003-09-25 04:51:05 GMT
\N
2016-11-04 04:29:31 GMT+5
2026-02-23 00:38:02 GMT+8
\N
2033-10-15 07:38:35 GMT+10
2003-07-29 03:55:56 GMT
1998-10-09 12:06:21 GMT-2
2006-04-02 07:46:45 GMT+1
1970-06-06 10:31:07 GMT-12
2015-02-06 12:13:46 GMT+4
2004-06-23 06:55:31 GMT+0
2015-12-08 00:58:12 GMT+4
2024-06-26 14:46:31 GMT+7
1997-11-07 07:28:15 GMT-2
\N
1992-11-16 02:10:16 GMT-4
\N
1996-11-09 19:05:11 GMT-3
2010-09-12 13:54:34 GMT+2
2012-10-17 04:03:22 GMT+3
2035-09-05 02:13:28 GMT+11
1998-07-19 07:49:42 GMT-2
1973-11-11 02:53:33 GMT-11
1992-11-15 01:08:33 GMT-4
2032-04-22 09:42:32 GMT+10
2031-09-14 17:56:28 GMT+10
2005-03-25 17:37:54 GMT+0
1993-03-17 02:02:52 GMT-4
2004-03-01 07:43:47 GMT+0
2009-10-06 23:07:36 GMT+2
1977-12-15 18:43:22 GMT-9
1993-06-07 21:43:23 GMT-4
\N
1987-08-07 06:17:36 GMT-6
2034-05-01 23:16:48 GMT+11
2025-06-19 19:55:50 GMT+8
2019-10-13 05:55:45 GMT+6
1977-03-25 16:49:59 GMT-9
1978-01-16 17:50:54 GMT-9
2003-09-29 11:52:36 GMT
\N
2024-01-12 08:48:56 GMT+7
\N
2036-08-16 06:28:01 GMT+11
1972-05-14 03:02:25 GMT-11
2019-01-24 15:28:12 GMT+5
2009-12-19 12:26:01 GMT+2
1982-07-26 20:02:07 GMT-8
2012-01-22 14:30:34 GMT+3
1983-07-22 09:43:31 GMT-7
1978-05-27 11:44:00 GMT-9
1974-04-02 20:06:06 GMT-11
2008-02-15 17:44:33 GMT+1
2003-01-14 11:53:21 GMT
1989-03-30 20:26:36 GMT-5
2009-12-14 07:04:27 GMT+2
\N
\N
\N
2031-07-02 05:37:29 GMT+10
2002-06-01 01:18:02 GMT-1
1980-07-28 23:17:09 GMT-8
2031-09-06 04:50:14 GMT+10
1975-01-20 00:56:52 GMT-10
\N
2025-05-02 21:09:49 GMT+8
\N
1991-09-27 13:58:05 GMT-4
\N
\N
2027-03-24 20:00:56 GMT+8
2009-01-07 11:43:43 GMT+2
1985-02-04 08:43:58 GMT-7
\N
1982-11-28 07:25:38 GMT-7
1996-12-23 15:44:40 GMT-2
2012-05-22 19:38:12 GMT+3
1980-02-29 23:26:57 GMT-8
2018-06-14 05:53:18 GMT+5
1981-07-04 12:51:41 GMT-8
1993-04-25 18:03:27 GMT-4
2027-03-09 22:57:21 GMT+8
1981-01-10 10:57:10 GMT-8
2005-11-25 19:03:21 GMT+1
1974-06-16 15:56:10 GMT-10
1985-10-15 14:41:42 GMT-6
\N
2030-12-31 21:55:54 GMT+10
1997-03-15 07:49:13 GMT-2
1982-02-26 20:08:21 GMT-8
1991-05-13 17:37:16 GMT-4
2015-09-03 11:31:02 GMT+4
1977-10-28 18:16:43 GMT-9
1987-06-05 08:47:52 GMT-6
1971-03-29 20:19:41 GMT-12
2002-03-28 08:10:03 GMT-1
2032-03-25 02:45:28 GMT+10
1983-07-23 15:43:13 GMT-7
2035-01-02 09:00:43 GMT+11
1999-01-15 18:02:24 GMT-2
2014-01-31 19:21:57 GMT+4
2021-09-02 07:24:41 GMT+6
1999-01-02 20:08:46 GMT-2
2001-03-24 22:19:29 GMT-1
2002-01-21 21:22:15 GMT-1
2007-08-06 07:40:41 GMT+1
1975-12-07 20:04:02 GMT-10
2032-09-07 01:44:27 GMT+10
1984-11-22 13:22:27 GMT-7
2035-06-18 23:03:59 GMT+11
1994-07-29 17:46:43 GMT-3
2023-04-20 23:06:32 GMT+7
2010-02-17 19:55:27 GMT+2
2014-08-23 04:24:35 GMT+4
2017-02-08 03:50:26 GMT+5
2036-10-24 01:43:48 GMT+12
1994-11-14 17:05:49 GMT-3
1979-04-19 00:51:15 GMT-9
1973-10-09 18:17:56 GMT-11
1999-01-17 10:01:57 GMT-2
2012-07-08 13:16:32 GMT+3
\N
1980-11-22 05:36:40 GMT-8
2002-02-28 09:35:09 GMT-1
2025-11-10 01:29:26 GMT+8
2005-03-24 03:31:43 GMT+0
\N
1987-09-11 07:03:58 GMT-6
\N
2026-01-29 10:21:54 GMT+8
2034-02-14 07:49:09 GMT+11
\N
\N
2031-10-25 07:54:40 GMT+10
2010-10-13 20:26:20 GMT+2
2036-09-11 09:50:27 GMT+12
2027-09-30 19:05:44 GMT+8
1995-04-10 19:58:12 GMT-3
1990-07-28 11:43:30 GMT-5
2001-05-05 00:25:16 GMT-1
2029-01-10 05:07:26 GMT+9
1972-10-09 13:38:21 GMT-11
2012-03-25 19:48:53 GMT+3
1987-04-06 01:11:45 GMT-6
1998-06-04 07:06:48 GMT-2
2038-01-16 08:56:45 GMT+12
2008-12-26 21:17:06 GMT+2
1981-08-23 10:06:56 GMT-8
1979-04-23 17:33:35 GMT-9
2010-01-16 23:22:58 GMT+2
1982-07-23 21:47:12 GMT-8
2013-08-21 20:50:30 GMT+3
1982-12-23 01:04:16 GMT-7
1997-04-03 19:25:47 GMT-2
1986-02-20 22:57:00 GMT-6
2034-09-08 09:15:33 GMT+11
1975-04-02 17:16:48 GMT-10
1999-05-14 02:29:36 GMT-2
1989-11-18 21:27:15 GMT-5
\N
2030-05-15 08:09:46 GMT+9
1978-05-03 02:10:34 GMT-9
2005-05-22 06:14:24 GMT+0
2001-02-11 14:04:25 GMT-1
1980-07-28 03:37:23 GMT-8
1984-04-11 21:36:37 GMT-7
2010-06-02 15:41:07 GMT+2
2023-09-05 03:40:25 GMT+7
2020-12-29 15:51:36 GMT+6
1997-06-06 05:10:09 GMT-2
2005-10-01 18:54:02 GMT+1
2023-01-17 16:49:50 GMT+7
2013-04-27 03:18:24 GMT+3
2018-05-23 23:13:58 GMT+5
2022-11-23 05:55:06 GMT+7
1971-05-09 22:21:51 GMT-12
\N
2008-08-22 07:12:18 GMT+2
2025-05-23 02:33:13 GMT+8
1989-02-18 10:23:41 GMT-5
2006-05-03 15:10:12 GMT+1
2035-08-21 05:44:10 GMT+11
1995-02-09 21:59:24 GMT-3
1994-11-28 00:22:35 GMT-3
1980-07-20 23:07:01 GMT-8
1998-09-02 22:25:59 GMT-2
1993-01-20 13:25:35 GMT-4
1979-12-21 08:44:13 GMT-8
\N
2022-10-19 07:06:12 GMT+7
\N
\N
2006-08-23 13:42:39 GMT+1
2024-05-31 02:55:37 GMT+7
2003-04-27 05:24:12 GMT
2029-03-21 01:01:47 GMT+9
2030-09-08 07:49:06 GMT+9
2022-05-05 19:07:56 GMT+6
1993-10-22 03:35:35 GMT-4
2031-04-01 07:28:30 GMT+10
2015-03-10 05:00:13 GMT+4
1988-12-08 16:27:26 GMT-5
\N
\N
2013-09-28 23:39:57 GMT+3
2030-02-25 07:31:16 GMT+9
2036-09-15 19:08:03 GMT+12
\N
2034-09-05 11:39:43 GMT+11
2003-11-07 21:07:04 GMT
\N
2002-09-19 07:59:10 GMT
2016-03-16 14:50:17 GMT+4
1986-02-06 02:57:29 GMT-6
2018-07-07 23:37:36 GMT+5
\N
2001-05-06 17:04:36 GMT-1
1996-10-05 13:24:45 GMT-3
1995-12-26 23:42:26 GMT-3
\N
1983-03-10 21:09:05 GMT-7
2011-08-03 07:47:28 GMT+3
2029-01-26 13:18:23 GMT+9
2022-04-24 00:45:21 GMT+6
1973-09-03 03:30:56 GMT-11
\N
1982-04-30 14:02:41 GMT-8
2023-11-30 23:11:51 GMT+7
1992-05-13 03:34:54 GMT-4
2001-11-13 00:47:25 GMT-1
1986-05-11 02:20:03 GMT-6
\N
\N
2022-02-08 01:26:47 GMT+6
1999-02-24 01:33:06 GMT-2
2007-09-30 10:55:32 GMT+1
1991-11-03 16:14:38 GMT-4
1985-06-14 21:54:21 GMT-7
1973-06-15 20:59:06 GMT-11
1972-07-18 12:26:22 GMT-11
\N
1989-01-17 11:48:58 GMT-5
1982-02-13 14:32:43 GMT-8
1994-11-23 11:36:17 GMT-3
2008-08-01 06:57:47 GMT+2
\N
1976-01-04 02:56:30 GMT-10
2037-04-11 12:30:59 GMT+12
2011-02-24 11:33:55 GMT+3
1992-11-16 04:53:09 GMT-4
2025-11-25 00:01:24 GMT+8
2013-02-08 02:14:43 GMT+3
2003-05-05 05:25:07 GMT
2002-05-13 14:26:05 GMT-1
1983-10-15 18:48:04 GMT-7
1995-01-07 19:03:41 GMT-3
1986-11-19 05:35:15 GMT-6
1991-04-09 21:07:05 GMT-4
2011-01-03 05:09:25 GMT+2
1974-12-06 19:39:43 GMT-10
2000-11-12 19:05:27 GMT-1
2006-01-31 04:14:52 GMT+1
2004-07-01 01:25:07 GMT+0
1978-06-04 03:51:35 GMT-9
\N
2002-07-29 17:13:57 GMT-1
1977-01-03 12:24:31 GMT-10
1991-12-12 18:57:03 GMT-4
1986-06-13 22:44:23 GMT-6
2012-12-08 18:42:47 GMT+3
\N
1977-10-01 22:10:45 GMT-9
1994-10-26 07:10:14 GMT-3
1997-01-11 02:26:04 GMT-2
2013-11-28 22:50:57 GMT+3
2016-07-14 09:05:32 GMT+4
\N
1994-10-27 20:12:10 GMT-3
2018-02-22 01:05:22 GMT+5
2034-10-05 12:42:38 GMT+11
2028-03-04 00:08:34 GMT+9
2037-05-21 23:24:07 GMT+12
2031-01-14 06:56:54 GMT+10
2036-04-03 10:15:05 GMT+11
2036-01-05 10:54:45 GMT+11
1979-09-22 14:31:26 GMT-9
\N
2027-08-13 08:28:38 GMT+8
2005-11-17 13:26:47 GMT+1
2028-02-23 18:32:03 GMT+9
1998-06-17 16:08:12 GMT-2
2024-03-02 13:53:12 GMT+7
1991-02-24 19:52:40 GMT-5
1973-11-23 22:00:24 GMT-11
2004-03-18 17:04:06 GMT+0
2033-08-13 19:45:25 GMT+10
2031-04-23 15:11:59 GMT+10
2009-12-31 01:57:02 GMT+2
1982-11-10 21:58:20 GMT-7
2010-10-07 04:13:26 GMT+2
1982-05-18 11:15:30 GMT-8
2010-09-28 14:05:51 GMT+2
1986-01-10 21:00:43 GMT-6
1997-03-20 05:04:03 GMT-2
2019-11-15 09:43:14 GMT+6
2017-09-06 00:45:47 GMT+5
2026-08-04 01:10:21 GMT+8
1992-01-20 08:03:40 GMT-4
2010-11-21 02:58:16 GMT+2
1974-03-27 16:19:28 GMT-11
2025-04-12 09:52:38 GMT+7
2014-09-14 22:11:22 GMT+4
1978-09-27 23:47:35 GMT-9
1985-10-29 21:31:25 GMT-6
2018-07-07 00:50:47 GMT+5
1989-08-31 01:29:41 GMT-5
1971-01-10 05:35:01 GMT-12
1983-05-28 15:52:19 GMT-7
2036-11-29 15:30:27 GMT+12
2022-01-24 17:56:31 GMT+6
2023-06-09 07:37:51 GMT+7
1989-06-08 04:26:44 GMT-5
2023-11-15 01:11:44 GMT+7
1997-04-16 06:19:03 GMT-2
2021-12-18 16:37:32 GMT+6
2009-10-07 21:30:12 GMT+2
1994-01-29 03:07:14 GMT-4
1991-05-26 12:45:10 GMT-4
1984-10-01 17:18:39 GMT-7
2034-01-04 01:22:45 GMT+11
1977-03-15 11:46:55 GMT-9
1990-11-22 14:57:16 GMT-5
2003-03-15 06:11:50 GMT
1977-08-27 08:26:42 GMT-9
1972-10-17 01:34:22 GMT-11
2032-09-15 22:54:13 GMT+10
1994-07-31 10:50:47 GMT-3
2024-08-25 05:13:02 GMT+7
2013-09-18 18:19:31 GMT+3
\N
\N
1981-12-25 22:21:59 GMT-8
\N
2008-10-31 04:45:33 GMT+2
2013-02-02 02:15:09 GMT+3
2018-10-18 12:56:11 GMT+5
1978-11-07 15:17:16 GMT-9
2037-01-19 01:13:16 GMT+12
\N
2021-04-21 08:45:28 GMT+6
2028-01-21 13:52:14 GMT+8
2015-03-11 16:07:57 GMT+4
2005-03-13 17:22:20 GMT+0
2004-01-28 20:45:12 GMT+0
1991-03-21 13:23:37 GMT-5
2029-07-26 06:01:08 GMT+9
infinity
infinity
infinity
infinity
infinity
infinity
2002-11-15 19:24:22 GMT
1987-05-02 16:25:01 GMT-6
1999-04-01 05:54:41 GMT-2
2034-03-18 02:35:05 GMT+11
1998-07-08 21:50:27 GMT-2
2015-06-08 05:27:05 GMT+4
1999-05-16 03:11:42 GMT-2
2017-10-29 15:46:32 GMT+5
2037-06-14 02:45:44 GMT+12
1997-11-29 16:30:09 GMT-2
1993-02-26 22:47:39 GMT-4
1998-12-04 16:43:34 GMT-2
2025-08-14 16:29:42 GMT+8
1982-07-31 16:54:13 GMT-8
\N
2029-04-26 12:37:56 GMT+9
2025-09-21 23:33:16 GMT+8
\N
2017-06-24 03:39:14 GMT+5
1990-11-17 19:15:50 GMT-5
2014-03-30 15:02:02 GMT+4
\N
2020-08-26 07:34:24 GMT+6
1992-11-14 14:14:38 GMT-4
2010-04-22 17:35:09 GMT+2
2034-03-23 18:10:28 GMT+11
1984-10-17 02:42:50 GMT-7
2001-04-29 03:59:54 GMT-1
1996-08-15 07:16:34 GMT-3
infinity
infinity
infinity
infinity
infinity
infinity
\N
1974-02-10 02:40:01 GMT-11
2033-03-28 21:51:56 GMT+10
2004-07-20 07:26:40 GMT+0
2017-07-25 17:26:58 GMT+5
2025-12-12 13:27:04 GMT+8
2021-10-01 01:30:40 GMT+6
1986-11-23 18:40:24 GMT-6
2025-10-28 20:22:31 GMT+8
2021-04-23 14:09:07 GMT+6
2002-03-22 12:22:14 GMT-1
2013-01-18 08:22:38 GMT+3
1988-06-29 03:31:44 GMT-5
1976-04-19 18:45:58 GMT-10
2008-01-09 11:25:14 GMT+1
1998-11-13 14:35:23 GMT-2
2033-12-18 16:43:16 GMT+11
1983-07-06 14:11:03 GMT-7
2017-07-23 00:18:04 GMT+5
2037-01-27 05:35:21 GMT+12
2004-09-14 14:29:32 GMT+0
2035-11-11 14:27:23 GMT+11
2021-05-09 15:44:42 GMT+6
1987-11-02 15:56:40 GMT-6
\N
2026-12-02 12:28:27 GMT+8
1990-12-27 22:26:41 GMT-5
\N
1987-07-11 01:54:54 GMT-6
2029-11-27 10:59:35 GMT+9
1990-04-16 21:11:47 GMT-5
2023-05-11 10:45:45 GMT+7
\N
2011-08-05 12:47:00 GMT+3
1993-06-09 06:44:41 GMT-4
1982-06-02 21:37:08 GMT-8
2025-08-13 13:03:45 GMT+8
1974-03-11 03:30:14 GMT-11
2011-09-27 19:47:44 GMT+3
1992-12-02 08:40:51 GMT-4
2033-10-11 14:20:32 GMT+10
1983-01-20 16:32:49 GMT-7
2002-09-15 02:56:52 GMT
2006-09-27 11:45:29 GMT+1
1997-06-29 05:54:17 GMT-2
\N
1984-02-22 06:27:16 GMT-7
1992-07-19 22:54:28 GMT-4
1991-12-22 22:41:11 GMT-4
1990-07-06 01:25:10 GMT-5
1973-06-15 06:34:08 GMT-11
1976-03-21 22:21:06 GMT-10
1976-06-15 07:14:46 GMT-10
-infinity
-infinity
-infinity
-infinity
-infinity
-infinity
1972-01-15 06:30:22 GMT-11
1971-08-06 10:41:43 GMT-11
1995-08-21 21:12:06 GMT-3
2006-07-30 07:14:27 GMT+1
2023-09-20 23:59:35 GMT+7
2003-03-18 13:53:03 GMT
2008-06-18 05:31:10 GMT+2
1993-06-07 06:55:33 GMT-4
2036-08-28 00:46:01 GMT+12
1970-05-11 08:36:31 GMT-12
\N
2019-01-10 06:01:34 GMT+5
1986-04-16 07:43:34 GMT-6
2007-07-14 07:46:45 GMT+1
2015-05-11 05:13:03 GMT+4
2025-12-09 22:39:17 GMT+8
\N
2012-11-18 03:02:25 GMT+3
2022-08-08 02:56:37 GMT+7
1988-10-20 06:54:55 GMT-5
1985-02-21 08:46:49 GMT-7
1996-05-22 09:18:01 GMT-3
1976-10-07 09:21:50 GMT-10
1981-08-22 18:22:24 GMT-8
2035-08-31 17:58:57 GMT+11
\N
\N
\N
2011-11-13 20:09:47 GMT+3
2018-04-02 08:00:33 GMT+5
1984-08-07 03:17:37 GMT-7
1988-07-13 10:31:02 GMT-5
\N
2029-08-28 23:46:32 GMT+9
2008-12-10 12:40:01 GMT+2
1974-01-10 13:11:31 GMT-11
1996-06-17 05:23:30 GMT-3
1974-06-05 10:42:56 GMT-10
1975-02-19 03:03:35 GMT-10
1994-06-29 10:30:45 GMT-3
2018-07-08 13:28:04 GMT+5
\N
1984-10-09 13:52:14 GMT-7
2033-12-23 18:53:07 GMT+11
2021-06-11 02:18:52 GMT+6
2027-01-20 17:50:31 GMT+8
2012-04-17 09:46:09 GMT+3
1982-03-08 23:06:30 GMT-8
2004-11-03 06:34:08 GMT+0
2028-05-30 05:51:58 GMT+9
1981-09-06 21:46:45 GMT-8
2004-08-12 16:29:39 GMT+0
1998-02-22 05:03:52 GMT-2
2023-03-08 13:32:55 GMT+7
1991-09-16 22:16:21 GMT-4
\N
1999-07-18 22:54:50 GMT-2
\N
1991-11-12 03:31:16 GMT-4
1993-08-28 02:05:54 GMT-4
2026-11-10 04:53:55 GMT+8
2015-06-21 03:00:10 GMT+4
2032-02-05 04:32:19 GMT+10
1974-11-25 19:29:07 GMT-10
1985-11-04 00:11:55 GMT-6
1992-11-23 19:38:16 GMT-4
1986-07-18 15:15:06 GMT-6
1987-10-25 00:29:12 GMT-6
1987-12-08 10:12:04 GMT-6
1996-11-06 14:53:07 GMT-3
2028-05-15 05:23:09 GMT+9
1998-12-12 15:18:10 GMT-2
1974-11-23 21:02:21 GMT-10
2021-08-17 04:11:28 GMT+6
2020-04-02 06:42:25 GMT+6
\N
1994-07-01 07:43:54 GMT-3
1989-03-09 09:10:29 GMT-5
2008-04-06 08:50:48 GMT+1
2027-02-23 20:30:36 GMT+8
1983-10-09 05:19:36 GMT-7
1995-10-16 11:47:58 GMT-3
2007-09-09 12:43:09 GMT+1
1977-12-16 00:24:08 GMT-9
2008-04-05 21:05:40 GMT+1
\N
1983-10-17 15:56:25 GMT-7
2029-08-12 12:45:17 GMT+9
1994-10-20 18:41:45 GMT-3
2017-12-08 16:48:31 GMT+5
2018-02-09 08:41:07 GMT+5
1973-12-13 16:00:22 GMT-11
2022-06-20 01:45:40 GMT+7
2033-05-25 10:33:11 GMT+10
2017-02-01 05:21:30 GMT+5
2019-02-01 20:00:39 GMT+5
1970-10-01 20:58:08 GMT-12
2025-01-17 02:23:47 GMT+7
1974-03-08 16:19:17 GMT-11
2020-05-27 21:01:29 GMT+6
1999-01-10 08:39:45 GMT-2
2009-05-26 10:18:22 GMT+2
2014-07-22 07:54:09 GMT+4
2036-10-26 20:20:46 GMT+12
2025-03-03 08:47:09 GMT+7
1972-09-22 21:58:10 GMT-11
1998-05-16 11:05:46 GMT-2
1994-09-14 08:08:42 GMT-3
1976-08-14 12:10:43 GMT-10
1975-03-14 19:29:02 GMT-10
1980-10-31 16:30:29 GMT-8
1975-10-03 19:29:30 GMT-10
2020-02-09 20:43:47 GMT+6
1975-04-02 10:00:13 GMT-10
2027-07-27 11:17:48 GMT+8
2022-05-23 09:28:31 GMT+6
1999-11-08 15:38:46 GMT-1
2007-01-14 02:48:02 GMT+1
2026-05-04 08:12:13 GMT+8
2029-06-19 09:20:46 GMT+9
2005-05-07 19:38:36 GMT+0
2023-04-23 14:15:53 GMT+7
2022-02-12 01:05:51 GMT+6
