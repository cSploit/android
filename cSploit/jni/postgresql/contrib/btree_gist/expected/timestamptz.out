-- timestamptz check
CREATE TABLE timestamptztmp (a timestamptz);
\copy timestamptztmp from 'data/timestamptz.data'
SET enable_seqscan=on;
SELECT count(*) FROM timestamptztmp WHERE a <  '2018-12-18 10:59:54 GMT+3';
 count 
-------
   391
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a <= '2018-12-18 10:59:54 GMT+3';
 count 
-------
   392
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a  = '2018-12-18 10:59:54 GMT+3';
 count 
-------
     1
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a >= '2018-12-18 10:59:54 GMT+3';
 count 
-------
   158
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a >  '2018-12-18 10:59:54 GMT+3';
 count 
-------
   157
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a <  '2018-12-18 10:59:54 GMT+2';
 count 
-------
   391
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a <= '2018-12-18 10:59:54 GMT+2';
 count 
-------
   391
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a  = '2018-12-18 10:59:54 GMT+2';
 count 
-------
     0
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a >= '2018-12-18 10:59:54 GMT+2';
 count 
-------
   158
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a >  '2018-12-18 10:59:54 GMT+2';
 count 
-------
   158
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a <  '2018-12-18 10:59:54 GMT+4';
 count 
-------
   392
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a <= '2018-12-18 10:59:54 GMT+4';
 count 
-------
   392
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a  = '2018-12-18 10:59:54 GMT+4';
 count 
-------
     0
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a >= '2018-12-18 10:59:54 GMT+4';
 count 
-------
   157
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a >  '2018-12-18 10:59:54 GMT+4';
 count 
-------
   157
(1 row)

SELECT a, a <-> '2018-12-18 10:59:54 GMT+2' FROM timestamptztmp ORDER BY a <-> '2018-12-18 10:59:54 GMT+2' LIMIT 3;
              a               |             ?column?              
------------------------------+-----------------------------------
 Tue Dec 18 05:59:54 2018 PST | @ 1 hour
 Thu Jan 10 03:01:34 2019 PST | @ 22 days 22 hours 1 min 40 secs
 Thu Jan 24 12:28:12 2019 PST | @ 37 days 7 hours 28 mins 18 secs
(3 rows)

CREATE INDEX timestamptzidx ON timestamptztmp USING gist ( a );
SET enable_seqscan=off;
SELECT count(*) FROM timestamptztmp WHERE a <  '2018-12-18 10:59:54 GMT+3'::timestamptz;
 count 
-------
   391
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a <= '2018-12-18 10:59:54 GMT+3'::timestamptz;
 count 
-------
   392
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a  = '2018-12-18 10:59:54 GMT+3'::timestamptz;
 count 
-------
     1
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a >= '2018-12-18 10:59:54 GMT+3'::timestamptz;
 count 
-------
   158
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a >  '2018-12-18 10:59:54 GMT+3'::timestamptz;
 count 
-------
   157
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a <  '2018-12-18 10:59:54 GMT+2'::timestamptz;
 count 
-------
   391
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a <= '2018-12-18 10:59:54 GMT+2'::timestamptz;
 count 
-------
   391
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a  = '2018-12-18 10:59:54 GMT+2'::timestamptz;
 count 
-------
     0
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a >= '2018-12-18 10:59:54 GMT+2'::timestamptz;
 count 
-------
   158
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a >  '2018-12-18 10:59:54 GMT+2'::timestamptz;
 count 
-------
   158
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a <  '2018-12-18 10:59:54 GMT+4'::timestamptz;
 count 
-------
   392
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a <= '2018-12-18 10:59:54 GMT+4'::timestamptz;
 count 
-------
   392
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a  = '2018-12-18 10:59:54 GMT+4'::timestamptz;
 count 
-------
     0
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a >= '2018-12-18 10:59:54 GMT+4'::timestamptz;
 count 
-------
   157
(1 row)

SELECT count(*) FROM timestamptztmp WHERE a >  '2018-12-18 10:59:54 GMT+4'::timestamptz;
 count 
-------
   157
(1 row)

EXPLAIN (COSTS OFF)
SELECT a, a <-> '2018-12-18 10:59:54 GMT+2' FROM timestamptztmp ORDER BY a <-> '2018-12-18 10:59:54 GMT+2' LIMIT 3;
                                     QUERY PLAN                                     
------------------------------------------------------------------------------------
 Limit
   ->  Index Scan using timestamptzidx on timestamptztmp
         Order By: (a <-> 'Tue Dec 18 04:59:54 2018 PST'::timestamp with time zone)
(3 rows)

SELECT a, a <-> '2018-12-18 10:59:54 GMT+2' FROM timestamptztmp ORDER BY a <-> '2018-12-18 10:59:54 GMT+2' LIMIT 3;
              a               |             ?column?              
------------------------------+-----------------------------------
 Tue Dec 18 05:59:54 2018 PST | @ 1 hour
 Thu Jan 10 03:01:34 2019 PST | @ 22 days 22 hours 1 min 40 secs
 Thu Jan 24 12:28:12 2019 PST | @ 37 days 7 hours 28 mins 18 secs
(3 rows)

